apis/beacon/blocks/attestations.yaml

```diff
@@ -0,0 +1,24 @@
+get:
+  operationId: getBlockAttestations
+  summary: Get block attestations
```

Comment:
Why this accessor specific for attestations? Is `/beacon/blocks/{block_id}` not sufficient?
And if this accessor, why not for the rest of the operations?

Reply:
Afaik there is a lot of use cases with just block attestation without rest of block data so it would reduce network traffic.

cc @mcdee @mkalinin

Reply:
I fine to merge if this was previously discussed